### ISSUE TITLE
Map object describes itself better

### ISSUE THREAD

**Issue by mweech** [fgpv-vpgf/fgpv-vpgf#1144]
When a user tabs or mouses over the map, it should describe itself so that it is obvious to someone using a screen reader that they are viewing a complex map (WCAG 2.0 AA 1.1.1)

**Comment by milesap** [comment]
Even with focus set on an element with an `aria-label`, the screen reader NVDA fails to read it, and instead says 'clickable' multiple time (presumably once for every single button on the page). This may require more formal training on how screen readers choose the content for audio output (or this is a bug with NVDA itself).

**Comment by mweech** [comment]
@milesap we have a license for JAWS that could be used as an alternative testing tool to nvda.
